tests/test_dependabot.py

```
@@ -0,0 +1,1342 @@
+            (),
+            {
+                "github_repo": "o/r",
+                "github_user": "u",
+                "dependabot": DependabotConfig(enabled=True),
+            },
+        )()
+        with (
+            patch("sova.config.loader.load_config", return_value=mock_cfg),
+            patch("sova.git.pr.list_open_prs", new_callable=AsyncMock, return_value=[]),
+        ):
+            await _maintain(project_dir=Path("/tmp/test"), dry_run=True)
```

Ruff (0.16.1)

[error] Probable insecure usage of temporary file or directory: "/tmp/test"

(S108)